xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -882,37 +884,37 @@ EDL::Action CEdl::GetLastEditActionType() const
   return m_lastEditActionType;
 }
 
-bool CEdl::GetNextSceneMarker(bool bPlus, const int iClock, int *iSceneMarker)
+bool CEdl::GetNextSceneMarker(Direction direction, int clock, int* sceneMarker)
```

Comment:
Would it be possible to replace the int pointer with a reference?

Reply:
I was not really interested in doing more in this PR but done in https://github.com/xbmc/xbmc/pull/25346/commits/0a4fb522bf2468697d9281c6c2480257cd309f19